.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```

Comment:
does this even work?
Seems like it has the wrong port etc?

Reply:
It works and launched on port 3000 

So should we just remove this?
We didn't add it

Reply:
yeah I think remove it - this file is meant to tell VS Code how to start it via the UI... but it looks like this won't work, so let's kill it! :)